phpBB/includes/acp/acp_board.php

```diff
@@ -623,6 +623,7 @@ function main($id, $mode)
 				}
 				else
 				{
+					send_status_line(403, 'Forbidden');
```

Comment:
`E_USER_ERROR` will result in a 503 being sent. This will mean we'll have duplicate Status header fields in this case.